config/routes.rb

```diff
@@ -11,6 +11,8 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
   get '/logout', to: 'sessions#destroy'
+  get '/dashboard', to: 'merchant/users#show'
```

Comment:
Don't we have these two paths from the scope operation on line 19?

Reply:
I think you're right here, too. I love these discoveries as they do not require us to do anything but delete something we don't need.

Reply:
You're right again, Peter. I don't think I had up to date routes on this branch so I wrote these in manually.